source/adapter.screenshare.js

```diff
@@ -0,0 +1,161 @@
+(function () {
+
+  'use strict';
+
+  var tempGetUserMedia = null;
+
+  if (window.navigator.mozGetUserMedia) {
+    tempGetUserMedia = window.navigator.getUserMedia;
+
+    window.navigator.getUserMedia = function (constraints, successCb, failureCb) {
+      constraints = constraints || {};
```

Comment:
The spec says that constraints is not optional and not nullable.
We should not have to default constraints this way.
https://w3c.github.io/mediacapture-main/getusermedia.html#dom-mediadevices-getusermedia